Dalamud/Interface/Internal/Windows/SelfTest/AgingSteps/MarketBoardAgingStep.cs

```diff
@@ -0,0 +1,259 @@
+                }
+                else
+                {
+                    ImGui.Text("Does one of the historical sales match this information?");
+                    ImGui.Separator();
+                    ImGui.Text($"Quantity: {this.historyListing.Quantity.ToString()}");
+                    ImGui.Text($"Buyer: {this.historyListing.BuyerName}");
+                    ImGui.Text($"Sale Price: {this.historyListing.SalePrice.ToString()}");
+                    ImGui.Text($"Purchase Time: {this.historyListing.PurchaseTime.ToString(CultureInfo.InvariantCulture)}");
+                    ImGui.Separator();
+                    if (ImGui.Button("Looks Correct"))
+                    {
+                        this.currentSubStep++;
+                    }
+
+                    ImGui.SameLine();
+                    if (ImGui.Button("No"))
+                    {
+                        return SelfTestStepResult.Fail;
+                    }
+                }
+
+                break;
+            case SubStep.Offerings:
+
+                if (this.itemListing == null)
+                {
+                    ImGui.Text("Goto a Market Board. Open any item that has sale listings.");
+                }
+                else
+                {
+                    ImGui.Text("Does one of the sales match this information?");
+                    ImGui.Separator();
+                    ImGui.Text($"Quantity: {this.itemListing.ItemQuantity.ToString()}");
+                    ImGui.Text($"Price Per Unit: {this.itemListing.PricePerUnit}");
+                    ImGui.Text($"Retainer Name: {this.itemListing.RetainerName}");
+                    ImGui.Text($"Is HQ?: {(this.itemListing.IsHq ? "Yes" : "No")}");
+                    ImGui.Separator();
+                    if (ImGui.Button("Looks Correct"))
+                    {
+                        this.currentSubStep++;
+                    }
+
+                    ImGui.SameLine();
+                    if (ImGui.Button("No"))
+                    {
+                        return SelfTestStepResult.Fail;
+                    }
+                }
+
+                break;
+            case SubStep.PurchaseRequests:
+                if (this.marketBoardPurchaseRequest == null)
+                {
+                    ImGui.Text("Goto a Market Board. Purchase any item, the cheapest you can find.");
+                }
+                else
+                {
+                    ImGui.Text("Does this information match the purchase you made? This is testing the request to the server.");
+                    ImGui.Separator();
+                    ImGui.Text($"Quantity: {this.marketBoardPurchaseRequest.ItemQuantity.ToString()}");
+                    ImGui.Text($"Item ID: {this.marketBoardPurchaseRequest.CatalogId}");
+                    ImGui.Text($"Price Per Unit: {this.marketBoardPurchaseRequest.PricePerUnit}");
+                    ImGui.Separator();
+                    if (ImGui.Button("Looks Correct"))
```

Comment:
Change label to `Looks Correct / Skip`. Buying things should not be necessary as part of the test step.